scgallery/designs/tiny_rocket/tiny_rocket.py

```
@@ -1,11 +1,249 @@
+    Returns:
+        List[Dict]: One entry per memory.
+    """
+    memories = []
+    with sc_open(path) as conf:
+        for line in conf:
+            fields = line.split()
+            if not fields:
+                continue
+            if fields[0] != 'name' or len(fields) % 2 != 0:
+                raise ValueError(f'unexpected memory configuration: {line.strip()}')
+            values = dict(zip(fields[::2], fields[1::2]))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)